src/App.tsx

```diff
@@ -15,36 +16,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Default = 'default',
+  Alphabetical = 'alphabetical',
+  Length = 'length',
+}
+
+interface TrackStatus {
```

Comment:
The name of the type is misleading.

Maybe `Filter`, `FilterProps`, `SortFields`, etc?